charts/victoria-metrics-agent/templates/service.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Service
+metadata:
+{{- if .Values.service.annotations }}
+  annotations:
+{{ toYaml .Values.service.annotations | indent 4}}
+{{- end }}
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+{{- if .Values.service.labels }}
```

Comment:
as you add labels in include, please rename `labels` to `extraLabels`

Reply:
fixed